### ISSUE TITLE
Google Drive issues

### ISSUE THREAD

**Issue by leongrdic** [buttercup/buttercup-mobile#272]
I came across two issues with Buttercup on my phone in regards to Google Drive.

First one is when I try to create a new archive, it gives me the following error but still creates the local vault:
![Screenshot_20211019_114430_com buttercup](https://user-images.githubusercontent.com/6260068/137887667-70a2e21a-b6c3-469e-86f7-ead51750b38e.jpg)
When I try to open the local archive I get the "possible tampering" error but when I check on google drive I can't find the archive file, meaning it wasn't created successfully.

Second issue is that I can't open archives that have a lot of entries in them. I tried creating an archive on google drive on desktop app and then importing passwords into it and only after there are entries I can no longer unlock the archive on my phone (unlocking empty archives on google drive works well). It's worth noting that the same archive (same count of entries) unlocks and works properly from WebDAV. I'm attaching screenshots of what happens:

![Screenshot_20211019_114320_com buttercup](https://user-images.githubusercontent.com/6260068/137888389-ca8c435d-eb33-4e91-86ad-782da310f7b2.jpg)

when I tap "use offline" it gets stuck loading like this and shows a new error:
![Screenshot_20211019_114323_com buttercup](https://user-images.githubusercontent.com/6260068/137888392-35ee0917-db0f-4501-bf2f-7c75f8df5ead.jpg)

Android 10 on Huawei VOG-L29

**Comment by Beef-Tech** [comment]
Hello,

Same issue for me, I imported all my Google Chrome passwords with the desktop App on a vault via Google Drive (it was long but ended successfully), and I'm trying to access to it via my phone (Android 11, Oppo Reno 4 Pro 5G).
![Screenshot_2021-12-27-00-11-50-82_6e85f0dfcd870fad08304e2ea1c47689](https://user-images.githubusercontent.com/62111944/147422204-7f1bae23-9a09-493c-9410-229df761f4ab.jpg)

Thanks !

**Comment by julianpoemp** [comment]
@leongrdic does the issue still exist with the latest version of Buttercup mobile (2.1.0) ?

**Comment by Beef-Tech** [comment]
Hello @julianpoemp 

I'm answering to my post because I had same issuers like @leongrdic (with all my Google Account passwords in the same directory to).

I've tried to re-open the vault after the update and now I can use it correctly.

I will post a new issue if necessary but actually it seems to be good !
